README.md

```diff
@@ -15,16 +15,19 @@ var token = '{YOUR_API_V2_TOKEN}';
 var auth0 = require('auth0')(token);
 ~~~
 
-### app_metadata
+### User's metadata
+Auth0 allows use to store data about a user that does not come from the user's identity provider. This is known as "metadata". There are two types of metadata: `user_metadata` and `app_metadata`. You can find out more [here](https://auth0.com/docs/apiv2Changes#8).
```

Comment:
We should create another doc I think, not just a link to apiv2 changes, anyway, for vnext :)